src/bt-hf-io.c

```diff
@@ -257,11 +314,16 @@ dtmf_cmd_cb(char tone)
 }
 
 static void
+#if ANDROID_VERSION >= 21
+nrec_cmd_cb(bthf_nrec_t nrec, bt_bdaddr_t* bd_addr)
+#else
 nrec_cmd_cb(bthf_nrec_t nrec)
+#endif
 {
   struct pdu_wbuf* wbuf;
 
-  wbuf = create_pdu_wbuf(1, /* NREC */
+  wbuf = create_pdu_wbuf(1 + /* NREC */
+                         6, /* address (BlueZ 5.25) */
```

Comment:
This variable is added since BlueZ 5.25, and the latest version of BlueZ is 5.27 now.

It seems there are at least two strategies now to comment the version number for this variable:
 - a. Use the lowest version that contains this variable.
 - b. Use the latest version that we adapted.

For strategy a, there is no out-date problem. The commented version is always correct and fixed.

For strategy b, these comments should be updated if we decide to use another version (ex. 5.27 or above) in the future. Since the version number of BlueZ went very fast last month, I am afraid we might need to update/upgrade our protocol version again soon.

What do you think?